src/components/Modal/index.web.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import withWindowDimensions from '../withWindowDimensions';
+import BaseModal from './BaseModal';
+import {propTypes, defaultProps} from './modalPropTypes';
+import * as StyleUtils from '../../styles/StyleUtils';
+
+const Modal = (props) => {
+    const setStatusBarColor = (color = '') => {
+        if (!props.fullscreen) { return; }
```

Comment:
```suggestion
        if (!props.fullscreen) {
            return;
        }
```
🤔 Not sure why the linter didn't catch this.

Reply:
The linter automatically fixed this from

```js
if (!props.fullscreen) return;
```

to this.